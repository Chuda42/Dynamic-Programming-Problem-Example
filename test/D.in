# dos reinicios
leer_datos
1 2 4 8 16 32 .
leer_rendimiento
32 16 8 4 2 1 .
max_datos_procesados
planificacion_optima
leer_datos
1 17 1 17 35 .
leer_rendimiento
30 5 4 3 2 .
max_datos_procesados
planificacion_optima
Fin
