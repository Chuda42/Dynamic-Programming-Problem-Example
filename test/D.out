1>2>3>4>Max Datos Procesados: 46
5>Cantidad Datos Planificacion: 46
6>7>8>Max Datos Procesados: 48
9>Cantidad Datos Planificacion: 48
10>Fin.
